components/Nav.js

```diff
@@ -8,7 +8,7 @@ import React from 'react'
 
 // Constants
 const links = {
-  Blog: {
+  News: {
```

Comment:
Please change this back to "blog" as it's still leading to the blog page. Unless you plan on refactoring half the website to turn "blog" into news, please don't reword just one part of it.

Reply:
Okay, understood